src/test/java/uk/gov/hmcts/reform/professionalapi/controller/internal/OrganisationInternalControllerTest.java

```diff
@@ -572,4 +574,29 @@ void testRetrieveOrgByPbaStatus() {
                 .getOrganisationsByPbaStatus(pbaStatus.toString());
     }
 
+    @Test
+    void testUpdateOrgName() {
+        final HttpStatus expectedHttpStatus = HttpStatus.OK;
+        String updatedName = "NewName";
+        organisation.setName(updatedName);
+        organisationNameUpdateRequest.setName(updatedName);
+
+        assertThat(organisationNameUpdateRequest.getName()).isNotEmpty();
```

Comment:
assertion seems redundant as we set the org name literally in previous statement above it